tests/agent/python/flaskapp/smoke.py

```diff
@@ -0,0 +1,257 @@
+import time
+from tornado import ioloop, httpclient
+from elasticsearch import Elasticsearch
+from datetime import datetime, timedelta
+import subprocess
+import logging
+import json
+import copy
+import uuid
+
+p_uid = uuid.uuid4().hex
+
+num_reqs = 0    # use with care
+
+logger = logging.getLogger("logger")
+logger.setLevel(logging.INFO)
+handler = logging.StreamHandler()
+handler.setLevel(logging.INFO)
+formatter = logging.Formatter('[%(asctime)s] [%(process)s] [%(levelname)s] [%(funcName)s - %(lineno)d]  %(message)s')
+handler.setFormatter(formatter)
+logger.propagate = False
+logger.addHandler(handler)
+
+es = Elasticsearch(['localhost:9200'])
+
+
+def handle(r):
+    global num_reqs
+    try:
+        assert r.code == 200
+        num_reqs -= 1
+        if num_reqs == 0:
+            logger.info("Stopping tornado I/O loop")
+            ioloop.IOLoop.instance().stop()
+
+    except AssertionError:
+        num_reqs == 0
+        ioloop.IOLoop.instance().stop()
+        logger.error("Bad response, aborting: {} - {} ({})".format(r.code, r.error, r.request_time))
+
+
+def run_process(lang, nworkers=4):
+    if lang == 'node':
+        cmd = ['node', '../../nodejs/app.js']
```

Comment:
We are at `tests/agent/python/flaskapp/smoke.py`. If this should be used as smoke test for both agents, could you please move it to a neutral directory?